Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -1572,6 +1572,9 @@ static float currentBacklightLevel()
 #if ENABLE(NOTIFY_BLOCKING)
 void WebProcess::postNotification(const String& message, std::optional<uint64_t> state)
 {
+    if (!WTF::processHasEntitlement("com.apple.developer.web-browser-engine.restrict.notifyd"_s))
```

Comment:
I don't get it. Why is it OK to ignore notifications when the entitlement is missing? Shouldn't we disable notification forwarding when the entitlement is missing? So that this code path is not even exercised?